Chrismarcel/iReporter
# #162395868 Fixes active menu link

__What does this PR do?__
This PR fixes the UI issue where the currently highlighted/active link does not reflect the current page the user is on.

__Description of tasks__
- Ensure that when users visit any of the pages, the currently selected/active menu link should reflect what page the user is on.

__How can this PR be tested__
- Visit https://chrismarcel.github.io/iReporter/UI
- Click on any on the menu links to be sent to a new page
- The menu link that was clicked on should be active and highlighted

__Link to relevant story__
[#162395868](https://www.pivotaltracker.com/story/show/162395868)
